src/pp.md

```diff
@@ -43,7 +43,7 @@ Open source
 [PP] is an Open source software.
 Anybody can contribute on [GitHub] to:
 
-- suggest or add new functionalities
+- suggest or add new functionality
```

Comment:
Because it's usually understood as being a "set":

> functionality: the set of functions or capabilities associated with computer software or hardware or an electronic device
>
> - https://www.merriam-webster.com/dictionary/functionality

It just sounds strange in plural... but maybe I'm wrong.